pkg-config isn't installed

### Summary

first time clone the repo && cd ladybird && .Meta/ladybird.py run
I get:
CMake Error at scripts/cmake/vcpkg_find_acquire_program.cmake:201 (message):
  Could not find pkg-config.  Please install it via your package manager: ...

### Operating system

macOS

### Steps to reproduce

1. git clone https://github.com/LadybirdBrowser/ladybird.git 
2. cd ladybird 
3.  .Meta/ladybird.py run


### Expected behavior

I'm not sure at this point (first time running) but I'm assuming it's starting up a build process to create the browser and I'm hoping it launches. I'll go ahead and install this "pkg-config" via homebrew manually.  But this bug needs documenting or directions to run "brew install pkg-config" before the other directions on the homepage.

### Actual behavior

broke.
I get:
CMake Error at scripts/cmake/vcpkg_find_acquire_program.cmake:201 (message):
  Could not find pkg-config.  Please install it via your package manager: ...

### URL for a reduced test case

N/A

### HTML/SVG/etc. source for a reduced test case

```html
N/A
```

### Log output and (if possible) backtrace


```shell
(#)~/Documents/code/ladybird master @ ./Meta/ladybird.py run
Cloning into 'vcpkg'...
remote: Enumerating objects: 309639, done.
remote: Counting objects: 100% (446/446), done.
remote: Compressing objects: 100% (170/170), done.
remote: Total 309639 (delta 384), reused 276 (delta 276), pack-reused 309193 (from 4)
Receiving objects: 100% (309639/309639), 97.90 MiB | 23.16 MiB/s, done.
Resolving deltas: 100% (208154/208154), done.
Checking out files: 100% (13569/13569), done.
Building vcpkg@05442024c3fda64320bd25d2251cc9807b84fb6f
Note: checking out '05442024c3fda64320bd25d2251cc9807b84fb6f'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 05442024c3 [openldap] fix emscripten build (#50088)
Downloading vcpkg-macos...
vcpkg package management program version 2025-12-16-44bb3ce006467fc13ba37ca099f64077b8bbf84d

See LICENSE.txt for license information.
Preset CMake variables:

  CMAKE_BUILD_TYPE="RelWithDebInfo"
  CMAKE_TOOLCHAIN_FILE="/Users/jlangley/Documents/code/ladybird/Meta/CMake/presets/../../../Build/vcpkg/scripts/buildsystems/vcpkg.cmake"
  LADYBIRD_CACHE_DIR="/Users/jlangley/Documents/code/ladybird/Meta/CMake/presets/../../../Build/caches"
  VCPKG_INSTALL_OPTIONS="--no-print-usage"
  VCPKG_OVERLAY_TRIPLETS="/Users/jlangley/Documents/code/ladybird/Meta/CMake/presets/../../../Meta/CMake/vcpkg/release-triplets"

Preset environment variables:

  LADYBIRD_SOURCE_DIR="/Users/jlangley/Documents/code/ladybird/Meta/CMake/presets/../../.."
  VCPKG_BINARY_SOURCES="clear;files,/Users/jlangley/Documents/code/ladybird/Meta/CMake/presets/../../../Build/caches/vcpkg-binary-cache,readwrite;"
  VCPKG_ROOT="/Users/jlangley/Documents/code/ladybird/Meta/CMake/presets/../../../Build/vcpkg"
  X_VCPKG_ASSET_SOURCES="clear;x-azurl,https://vcpkg-cache.app.ladybird.org/ladybird/source-assets/,,read"

-- Determined host VCPKG_TARGET_TRIPLET: arm64-osx-dynamic
-- Running vcpkg install
A suitable version of cmake was not found (required v3.31.10).
Trying to download cmake-3.31.10-macos-universal.tar.gz using asset cache https://vcpkg-cache.app.ladybird.org/ladybird/source-assets/5960326dee8227bf27cdd2d94c336835b7bf1b11c443c35ecf2b50a811c7874fcc3400a3b793e4720ef6505d66aed597ecdaf6c77b12db69abafcccd0659182d
Asset cache miss; trying authoritative source https://github.com/Kitware/CMake/releases/download/v3.31.10/cmake-3.31.10-macos-universal.tar.gz
Successfully downloaded cmake-3.31.10-macos-universal.tar.gz
Extracting cmake...
Detecting compiler hash for triplet arm64-osx-dynamic...
Compiler found: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++
The following packages will be built and installed:
  * abseil:arm64-osx-dynamic@20250814.1
    angle[core,metal]:arm64-osx-dynamic@chromium_7258 -- /Users/jlangley/Documents/code/ladybird/Meta/CMake/vcpkg/overlay-ports/angle
  * brotli:arm64-osx-dynamic@1.2.0
  * bzip2[core,tool]:arm64-osx-dynamic@1.0.8#6
    cpptrace:arm64-osx-dynamic@1.0.2 -- /Users/jlangley/Documents/code/ladybird/Meta/CMake/vcpkg/overlay-ports/cpptrace
    curl[brotli,core,http2,http3,non-http,openssl,ssl,websockets,zstd]:arm64-osx-dynamic@8.18.0#1
  * dav1d:arm64-osx-dynamic@1.5.1
  * dirent:arm64-osx-dynamic@1.26
  * egl-registry:arm64-osx-dynamic@2025-05-27
  * expat:arm64-osx-dynamic@2.7.4
    fast-float:arm64-osx-dynamic@8.1.0
    ffmpeg[avcodec,avdevice,avfilter,avformat,core,dav1d,openh264,opus,swresample,swscale,theora,vorbis,vpx,webp,zlib]:arm64-osx-dynamic@7.1.1#5
    fmt:arm64-osx-dynamic@12.1.0
    fontconfig:arm64-osx-dynamic@2.15.0#4
  * freetype[brotli,bzip2,core,png,zlib]:arm64-osx-dynamic@2.13.3
  * gettext:arm64-osx-dynamic@0.22.5#4
  * gettext-libintl:arm64-osx-dynamic@0.22.5#4
  * giflib:arm64-osx-dynamic@5.2.2#2
  * gperf:arm64-osx-dynamic@3.3
    harfbuzz[core,coretext,freetype,icu]:arm64-osx-dynamic@10.2.0
  * highway:arm64-osx-dynamic@1.3.0
    icu[core,tools]:arm64-osx-dynamic@78.1#1
  * lcms:arm64-osx-dynamic@2.18
    libavif[core,dav1d]:arm64-osx-dynamic@1.3.0#1
    libdwarf:arm64-osx-dynamic@2.3.0
  * libiconv:arm64-osx-dynamic@1.18#3
    libjpeg-turbo:arm64-osx-dynamic@3.1.1
    libjxl:arm64-osx-dynamic@0.11.1#3
  * liblzma:arm64-osx-dynamic@5.8.2
  * libogg:arm64-osx-dynamic@1.3.6#1
    libpng[apng,core]:arm64-osx-dynamic@1.6.50
    libproxy:arm64-osx-dynamic@0.4.18#3
  * libtheora:arm64-osx-dynamic@1.2.0
    libtommath:arm64-osx-dynamic@1.3.0#2 -- /Users/jlangley/Documents/code/ladybird/Meta/CMake/vcpkg/overlay-ports/libtommath
  * libvorbis:arm64-osx-dynamic@1.3.7#4
  * libvpx:arm64-osx-dynamic@1.16.0
    libwebp[anim,core,img2webp,libwebpmux,mux,nearlossless,simd]:arm64-osx-dynamic@1.6.0#1
    libxml2[core,iconv,zlib]:arm64-osx-dynamic@2.13.8#1
  * libyuv:arm64-osx-dynamic@1916
  * nghttp2:arm64-osx-dynamic@1.68.0#1 -- /Users/jlangley/Documents/code/ladybird/Meta/CMake/vcpkg/overlay-ports/nghttp2
  * nghttp3:arm64-osx-dynamic@1.15.0
  * ngtcp2[core,openssl]:arm64-osx-dynamic@1.20.0
  * opengl-registry:arm64-osx-dynamic@2026-01-26
  * openh264:arm64-osx-dynamic@2.6.0#4
    openssl:arm64-osx-dynamic@3.5.3
  * opus:arm64-osx-dynamic@1.5.2#1
  * pkgconf:arm64-osx-dynamic@2.5.1#4
  * pthread:arm64-osx-dynamic@3.0.0#2
  * pthreads:arm64-osx-dynamic@3.0.0#14
    sdl3:arm64-osx-dynamic@3.2.28
    simdjson[core,deprecated,exceptions,threads,utf8-validation]:arm64-osx-dynamic@4.2.4
    simdutf:arm64-osx-dynamic@7.4.0
    skia[core,fontconfig,freetype,harfbuzz,icu,metal]:arm64-osx-dynamic@144
    sqlite3[core,json1]:arm64-osx-dynamic@3.50.4
    tiff[core,jpeg,lzma,zip,zstd]:arm64-osx-dynamic@4.7.1
  * vcpkg-cmake:arm64-osx-dynamic@2024-04-23
  * vcpkg-cmake-config:arm64-osx-dynamic@2024-05-23
  * vcpkg-cmake-get-vars:arm64-osx-dynamic@2025-05-29
  * vcpkg-get-python-packages:arm64-osx-dynamic@2025-04-05
  * vcpkg-gn:arm64-osx-dynamic@2025-08-05
  * vcpkg-make:arm64-osx-dynamic@2026-01-01
  * vcpkg-pkgconfig-get-modules:arm64-osx-dynamic@2024-04-03
  * vcpkg-tool-gn:arm64-osx-dynamic@2025-08-05#1
  * vcpkg-tool-meson:arm64-osx-dynamic@1.9.0#3
    vulkan:arm64-osx-dynamic@2023-12-17 -- /Users/jlangley/Documents/code/ladybird/Meta/CMake/vcpkg/overlay-ports/vulkan
    vulkan-headers:arm64-osx-dynamic@1.4.335.0
    woff2:arm64-osx-dynamic@1.0.2#5
    zlib:arm64-osx-dynamic@1.3.1
  * zstd:arm64-osx-dynamic@1.5.7
Additional packages (*) will be modified to complete this operation.
Restored 0 package(s) from /Users/jlangley/Documents/code/ladybird/Meta/CMake/presets/../../../Build/caches/vcpkg-binary-cache in 680 us. Use --debug to see more details.
Installing 1/69 vcpkg-cmake:arm64-osx-dynamic@2024-04-23...
Building vcpkg-cmake:arm64-osx-dynamic@2024-04-23...
/Users/jlangley/Documents/code/ladybird/Meta/CMake/vcpkg/release-triplets/arm64-osx-dynamic.cmake: info: loaded overlay triplet from here
-- Installing: /Users/jlangley/Documents/code/ladybird/Build/vcpkg/packages/vcpkg-cmake_arm64-osx-dynamic/share/vcpkg-cmake/vcpkg_cmake_configure.cmake
-- Installing: /Users/jlangley/Documents/code/ladybird/Build/vcpkg/packages/vcpkg-cmake_arm64-osx-dynamic/share/vcpkg-cmake/vcpkg_cmake_build.cmake
-- Installing: /Users/jlangley/Documents/code/ladybird/Build/vcpkg/packages/vcpkg-cmake_arm64-osx-dynamic/share/vcpkg-cmake/vcpkg_cmake_install.cmake
-- Installing: /Users/jlangley/Documents/code/ladybird/Build/vcpkg/packages/vcpkg-cmake_arm64-osx-dynamic/share/vcpkg-cmake/vcpkg-port-config.cmake
-- Installing: /Users/jlangley/Documents/code/ladybird/Build/vcpkg/packages/vcpkg-cmake_arm64-osx-dynamic/share/vcpkg-cmake/copyright
-- Performing post-build validation
Starting submission of vcpkg-cmake:arm64-osx-dynamic@2024-04-23 to 1 binary cache(s) in the background
Elapsed time to handle vcpkg-cmake:arm64-osx-dynamic: 85.2 ms
vcpkg-cmake:arm64-osx-dynamic package ABI: a1a28e76a81fdd3ddbc7efe333062725d0e4258dde15e1c18b68d68d6c0b0f51
Installing 2/69 zlib:arm64-osx-dynamic@1.3.1...
Building zlib:arm64-osx-dynamic@1.3.1...
/Users/jlangley/Documents/code/ladybird/Meta/CMake/vcpkg/release-triplets/arm64-osx-dynamic.cmake: info: loaded overlay triplet from here
Trying to download madler-zlib-v1.3.1.tar.gz using asset cache https://vcpkg-cache.app.ladybird.org/ladybird/source-assets/8c9642495bafd6fad4ab9fb67f09b268c69ff9af0f4f20cf15dfc18852ff1f312bd8ca41de761b3f8d8e90e77d79f2ccacd3d4c5b19e475ecf09d021fdfe9088
Download successful! Asset cache hit, did not try authoritative source https://github.com/madler/zlib/archive/v1.3.1.tar.gz
-- Extracting source /Users/jlangley/Documents/code/ladybird/Build/vcpkg/downloads/madler-zlib-v1.3.1.tar.gz
-- Applying patch 0001-Prevent-invalid-inclusions-when-HAVE_-is-set-to-0.patch
-- Applying patch 0002-build-static-or-shared-not-both.patch
-- Applying patch 0003-android-and-mingw-fixes.patch
-- Using source at /Users/jlangley/Documents/code/ladybird/Build/vcpkg/buildtrees/zlib/src/v1.3.1-2e5db616bf.clean
-- Configuring arm64-osx-dynamic
-- Building arm64-osx-dynamic-rel
-- Installing: /Users/jlangley/Documents/code/ladybird/Build/vcpkg/packages/zlib_arm64-osx-dynamic/share/zlib/vcpkg-cmake-wrapper.cmake
-- Fixing pkgconfig file: /Users/jlangley/Documents/code/ladybird/Build/vcpkg/packages/zlib_arm64-osx-dynamic/lib/pkgconfig/zlib.pc
CMake Error at scripts/cmake/vcpkg_find_acquire_program.cmake:201 (message):
  Could not find pkg-config.  Please install it via your package manager:

      brew install pkg-config
Call Stack (most recent call first):
  scripts/cmake/vcpkg_fixup_pkgconfig.cmake:193 (vcpkg_find_acquire_program)
  buildtrees/versioning_/versions/zlib/3f05e04b9aededb96786a911a16193cdb711f0c9/portfile.cmake:43 (vcpkg_fixup_pkgconfig)
  scripts/ports.cmake:206 (include)


error: building zlib:arm64-osx-dynamic failed with: BUILD_FAILED
See https://learn.microsoft.com/vcpkg/troubleshoot/build-failures?WT.mc_id=vcpkg_inproduct_cli for more information.
Elapsed time to handle zlib:arm64-osx-dynamic: 8.1 s
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+zlib
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?title=%5Bzlib%5D%20build%20error%20on%20arm64-osx-dynamic&body=Copy%20issue%20body%20from%20%2FUsers%2Fjlangley%2FDocuments%2Fcode%2Fladybird%2FBuild%2Frelease%2Fvcpkg_installed%2Fvcpkg%2Fissue_body.md
You can also submit an issue by running (GitHub CLI must be installed):
  /usr/local/bin/gh issue create -R microsoft/vcpkg --title "[zlib] Build failure on arm64-osx-dynamic" --body-file /Users/jlangley/Documents/code/ladybird/Build/release/vcpkg_installed/vcpkg/issue_body.md
Completed submission of vcpkg-cmake:arm64-osx-dynamic@2024-04-23 to 1 binary cache(s) in 153 ms
-- Running vcpkg install - failed
CMake Error at Build/vcpkg/scripts/buildsystems/vcpkg.cmake:953 (message):
  vcpkg install failed.  See logs for more information:
  /Users/jlangley/Documents/code/ladybird/Build/release/vcpkg-manifest-install.log
Call Stack (most recent call first):
  /usr/local/share/cmake-3.31/Modules/CMakeDetermineSystem.cmake:146 (include)
  CMakeLists.txt:26 (project)


CMake Error: CMake was unable to find a build program corresponding to "Ninja".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
-- Configuring incomplete, errors occurred!
```

### Screenshots or screen recordings

_No response_

### Build flags or config settings

_No response_

### Contribute a patch?

- [ ] I’ll contribute a patch for this myself.

--- a/src/components/landing/get-involved.astro
+++ b/src/components/landing/get-involved.astro
@@ -3,12 +3,19 @@
     <h2 class="section-heading">Get involved</h2>
     <p>
       We welcome new contributors every week across all areas of the engine.
-      Clone the code, build it, and come say hi.
+      Clone the code, build it, and come say hi. On many development machines it
+      is as simple as this:
     </p>
 
     <pre><code>$ git clone https://github.com/LadybirdBrowser/ladybird.git
 $ cd ladybird
 $ ./Meta/ladybird.py run</code></pre>
+    <p>
+      For more detailed, operating-system-specific information, check out our <a
+        href="https://github.com/LadybirdBrowser/ladybird/tree/master/Documentation/BuildInstructionsLadybird.md"
+        >full build instructions</a
+      >.
+    </p>
 
     <p>
       <strong>Source code:</strong>

